recipes/service.rb

```diff
@@ -74,17 +74,35 @@
   not_if { node['splunk']['server']['runasroot'] }
 end
 
-template '/etc/init.d/splunk' do
-  source 'splunk-init.erb'
-  mode 0700
-  variables(
-    splunkdir: splunk_dir,
-    runasroot: node['splunk']['server']['runasroot']
-  )
-end
+if node['init_package'] == 'systemd'
```

Comment:
Just FYI this will require we increase the minimum supported Chef version to 11.12 where we introduced this attribute. I'm ok with that, but we'll need to update the readme